### DISCUSSION TITLE
Official Python 3.10 Release

### DISCUSSION THREAD

**Discussion by PavelPancocha** [geopy/geopy#522] [Help] [answered]
Hello, 
when is the official Python 3.10 release planned? 

I see, it works fine, but official support would be great!

Thank you!

Pavel

**Comment by KostyaEsmukov** [comment, marked as answer]
Hi,

It's already in master (40717e64e9a79ab52ecbf39530f7c5d663c5d664) and will be released as 2.3.0. Can't promise any release date though.
